.cfnlintrc.yaml

```diff
@@ -147,3 +148,5 @@ ignore_checks:
   - E3001 # Invalid or unsupported Type; common in transform tests since they focus on SAM resources
   - W2001 # Parameter not used
   - E3006 # Resource type check; we have some Foo Bar resources
+  - W3037 # Ignore cfn-lint check for non existing IAM permissions
+  - E3510 # Ignore cfn-lint check for non existing IAM policies
```

Comment:
I agree with Frederic's comment, but also, we likely only need `W3037` (according to the [tests that are failing](https://github.com/aws/serverless-application-model/actions/runs/13912627739/job/38929834041))

Reply:
There were other tests failing for `E3510`.